tests/e2e/core-tests/specs/merchant/wp-admin-order-status-filters.test.js

```diff
@@ -57,7 +57,7 @@ const runOrderStatusFiltersTest = () => {
 			await createSimpleOrder(orderStatus.cancelled.description.text);
 			await createSimpleOrder(orderStatus.refunded.description.text);
 			await createSimpleOrder(orderStatus.failed.description.text);
-		}, 40000);
+		}, 60000);
```

Comment:
I wonder if we should look at using `Batch update orders` to create orders like this (I believe the API will allow passing in a status, if I recall correctly) to not only avoid flakiness and needing to up the timeout, but also make the test more efficient.

Reply:
Yes, this should be added as one of the tasks to complete after #27873.

Reply:
Sounds good--I created https://github.com/woocommerce/woocommerce/issues/29797 to follow up on updating these tests to use the API once it's added.